prow/plugins/mergecommitblocker/mergecommitblocker_test.go

```diff
@@ -0,0 +1,124 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package mergecommitblocker
+
+import (
+	"fmt"
+	"github.com/sirupsen/logrus"
+	"k8s.io/test-infra/prow/github"
+	"k8s.io/test-infra/prow/github/fakegithub"
+	"reflect"
+	"testing"
+)
+const (
+	helpWanted = "help-wanted"
+	mergeCommit = "do-not-merge/contains-merge-commits"
```

Comment:
Why not import this?

Reply:
i will admit i was copying part of another plugins tests 😎will fix